dateutil/test/test_parser.py

```diff
@@ -27,6 +30,24 @@
 except ValueError:
     PLATFORM_HAS_DASH_D = False
 
+class TestConvertyear(unittest.TestCase):
+    @given(integers(min_value=100))
+    def test_convertyear(self, n):
+        self.assertEqual(n, _parser.parserinfo().convertyear(n))
+
+    @given(integers(min_value=-50,
+                    max_value=49))
+    def test_no_specified_century(self, n):
+        p = _parser.parserinfo()
+        new_year = p._year + n
+        result = p.convertyear(new_year % 100, century_specified=False)
+        self.assertEqual(result, new_year)
```

Comment:
@pganssle is there a preferred convention for how to do these assertions?  I tend to prefer just `assert result == new_year`

Reply:
@jbrockmendel Yeah, these should be refactored as pytest tests. This was part of a sprint event and I told @coreygirard that it was fine to write them as UnitTest tests and I'll refactor them into pytest tests before merging.